docs/tutorials/audio.ipynb

```diff
@@ -0,0 +1,475 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "wtM_ixN724xb",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "audio_slice = audio[100:]\n",
+        "\n",
+        "# remove last dimension\n",
+        "audio_tensor = tf.squeeze(audio_slice, axis=[-1])\n",
+        "\n",
+        "print(audio_tensor)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fmt4cn304IbG",
+        "colab_type": "text"
+      },
+      "source": [
+        "It is more convinient to convert tensor into float numbers and show the audio clip in graph:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ZpwajOeR4UMU",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import matplotlib.pyplot as plt\n",
+        "\n",
+        "\n",
+        "tensor = tf.cast(audio_tensor, tf.float32) / 32768.0\n",
+        "\n",
+        "plt.figure()\n",
+        "plt.plot(tensor.numpy())"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "86qE8BPl5rcA",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Trim the noise\n",
+        "\n",
+        "Sometimes it makes sense to trim the noise from the audio, which could be done through API `tfio.experimental.audio.trim`. Returned from the API is a pair of `[start, stop]` position of the segement:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "eEa0Z5U26Ep3",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "position = tfio.experimental.audio.trim(tensor, axis=0, epsilon=0.1)\n",
+        "print(position)\n",
+        "\n",
+        "start = position[0]\n",
+        "stop = position[1]\n",
+        "print(start, stop)\n",
+        "\n",
+        "processed = tensor[start:stop]\n",
+        "\n",
+        "plt.figure()\n",
+        "plt.plot(processed.numpy())"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ineBzDeu-lTh",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Fade In and Fade Out\n",
+        "\n",
+        "One useful audio engineering technique is fade, which gradually increases or decreases audio signals. This can be done through `tfio.experimental.audio.fade`. `tfio.experimental.audio.fade` supports different shapes of fades such as `linear`, `logarithmic`, or `exponential`:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "LfZo0XaaAaeM",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "fade = tfio.experimental.audio.fade(\n",
```

Comment:
Optional

See:  https://www.tensorflow.org/hub/tutorials/spice

You can use `from IPython.display import Audio` to embed the audio as a little playable widget.

Reply:
This looks really nice 👍 I have added the widget.